index/scorch/snapshot_index.go

```diff
@@ -424,6 +435,11 @@ func (i *IndexSnapshot) Document(id string) (rv index.Document, err error) {
 	segmentIndex, localDocNum := i.segmentIndexAndLocalDocNumFromGlobal(docNum)
 
 	rvd := document.NewDocument(id)
+	var prevBytesRead uint64
+	seg, ok := i.segment[segmentIndex].segment.(segment.DiskStatsReporter)
```

Comment:
Rename ok to say diskStatsAvailable so in line 472 below we know why things are ok.